enterprise/backend/test/metabase_enterprise/sandbox/api/gtap_test.clj

```diff
@@ -216,8 +216,9 @@
                               :card_id  card-id-1))))
 
           (testing "Test that we can update a sandbox using the permission graph API"
-            (let [sandbox-id (db/select-one-field :id GroupTableAccessPolicy {:table_id table-id-1
-                                                                              :group_id group-id})
+            (let [sandbox-id (db/select-one-field :id GroupTableAccessPolicy
+                                                  :table_id table-id-1
+                                                  :group_id group-id)
```

Comment:
The fact that this ever worked in Toucan 1 was a pure accident, and I'm 95% sure it was unintentional, since there were no tests around it, it was undocumented, and I was totally confused to see this code in real life. Updated to an equivalent form that won't break when we switch to Toucan 2.